base_layer/core/src/block.rs

```diff
@@ -67,10 +67,11 @@ impl Block {
 }
 
 // todo this probably need to move somewhere else
-/// This function will create the correct amount for the coinbase given the block height
+/// This function will create the correct amount for the coinbase given the block height, it will provide the answer in
```

Comment:
See `emission.rs` - let's incorporate that sooner rather than later

Reply:
Going to address this in a next pr refactoring u64 to MicroTari for amounts